test/setup-ca-kubernetes.sh

```diff
@@ -41,21 +39,24 @@ EOF
 
 
 # Directory to use for storing intermediate files
-WORKDIR=${WORKDIR:-/tmp/certs}
+WORKDIR=${WORKDIR:-$(mktemp -d -u -t pmem-XXXX)}
 mkdir -p $WORKDIR 2> /dev/null
```

Comment:
I think mkdir -p writes no output in normal case, and if it does it's likely serious enough to be shown, message may be helpful as following script part likely will fail.
I would take away redirection to /dev/null here.

Reply:
I agree, will change this.

Reply:
Done